### PR TITLE
Update view & i18n helper. Add new locale zh-TW

### PR DESCRIPTION
**Pull request by imgarylai**
This is relevant to #122 & #139 

- Updated the i18n helpers to use the latest Spree i18n helper. 
- Add new locale `zh-TW`
- Tweak a little bit of the UI for the latest spree-backend theme

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -48,3 +52,10 @@ en:
       invoice_slip_legend: Invoice/Slip Content
       invoice_number: 'Invoice number: %{number}'
       invoice_date: 'Invoice date: %{date}'
+      date: Date
+      number: Number
+      firstname: First name
+      lastname: Last name
+      document_type: Document type
+      email: Email
+      total: Total
```
Comment (voleoo) [inline config/locales/en.yml:61]: Please add empty line at the end of file.
Also in config/locales/zh-TW.yml

Comment (voleoo) [reply config/locales/en.yml:61]: What about `app/views/spree/admin/orders/_print_invoice_order_tab_links.html.erb`
I think this file also should have small changes 
```
<% if can? :show, Spree::Order %>
  <% if @order && @order.completed_at? %>
      <li data-hook="admin_order_tabs_documents">
        <%=
          link_to_with_icon 'file',
            Spree.t(:documents, scope: :print_invoice),
            spree.admin_order_bookkeeping_documents_path(@order),
            title: title,
            class: "#{'active ' if current == :documents}nav-link icon-link with-tip action-refresh"
        %>
      </li>
  <% end %>
<% end %>
```

**Comment by voleoo** [comment]
What about `app/views/spree/admin/orders/_print_invoice_order_tab_links.html.erb`
I think this file also should have small changes 
```
<% if can? :show, Spree::Order %>
  <% if @order && @order.completed_at? %>
      <li data-hook="admin_order_tabs_documents">
        <%=
          link_to_with_icon 'file',
            Spree.t(:documents, scope: :print_invoice),
            spree.admin_order_bookkeeping_documents_path(@order),
            title: title,
            class: "#{'active ' if current == :documents}nav-link icon-link with-tip action-refresh"
        %>
      </li>
  <% end %>
<% end %>
```

**Comment by voleoo** [comment]
And please change name of PR, it's not only about locales

**Comment by imgarylai** [comment]
Thank you @voleoo 

I have updated this PR.

File: app/views/spree/admin/bookkeeping_documents/index.html.erb [hunk 2]
```diff
@@ -86,24 +86,24 @@
 <% end %>
 
 <% if @bookkeeping_documents.any? %>
-  <table class="table" id="listing_orders" data-hook>
+  <table class="table table-responsive" id="listing_orders" data-hook>
```
Comment (voleoo) [inline app/views/spree/admin/bookkeeping_documents/index.html.erb:89]: could you use `table-responsive-lg` instead `table-responsive`

**Comment by imgarylai** [comment]
@voleoo Thanks for your review. I made the update and pushed it to GitHub.

**Comment by voleoo** [comment]
It's strange that the github failed, but TravisCI is ok.

@damianlegawiec  I think it's should be merged to master for support Spree > 4
Thanks @imgarylai

**Pull request merged**